recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ source:
     folder: testing
```

Comment:
Please update the testing submodule, to the commit used as of https://github.com/apache/arrow/tree/apache-arrow-15.0.0

Reply:
Thanks! I didn't even notice we had that version there. I am learning a lot :)

Reply:
The reason is pretty straightforward - we want to run the test suite, which requires the testing submodule, but it is not part of the tarball, so we need to pull it in (and keep it in sync) manually :)